src/ui/assets/AssetsUploadContainer.js

```diff
@@ -17,7 +16,13 @@ export const mapDispatchToProps = dispatch => ({
       })),
     };
     dispatch(setInfo(assetInfo));
-    dispatch(setVisibleModal(UPLOAD_ASSET_MODAL_ID));
+    dispatch(setVisibleModal(`${UPLOAD_ASSET_MODAL_ID}${name}`));
+  },
+  onAssetSelected: (asset) => {
```

Comment:
Please correct me if I'm wrong, but this function is called when the upload is started. What about `onUploadStarted`, `onUpload` or something like that?